tsung.xml

```diff
@@ -61,56 +64,79 @@
+               <request subst="true">
+                   <!-- Skillz message format with randomized message and id. -->
+                   <jabber type="raw" ack="no_ack" data="&lt;message xmlns=&quot;jabber:client&quot; to=&quot;81@conference.chat.staging.skillz.com&quot; id=&quot;%%ts_user_server:get_unique_id%%&quot; type=&quot;groupchat&quot;&gt; &lt;body&gt;%%_random_message%%&lt;/body&gt; &lt;skillz_sdk xmlns=&quot;xmpp:skillz&quot;&gt; &lt;avatar_url&gt;https://cdn.staging.skillz.com/profile-pics/thumbnail/c11828ac-38b9-4b60-a701-8cd4d6d57c49&lt;/avatar_url&gt; &lt;flag_url&gt;https://cdn.staging.skillz.com/flags/US.png&lt;/flag_url&gt; &lt;user_id&gt;%%_username%%&lt;/user_id&gt; &lt;username&gt;%%_username%%&lt;/username&gt; &lt;user_role&gt;0&lt;/user_role&gt; &lt;user_mentions&gt;[]&lt;/user_mentions&gt; &lt;message_type&gt;0&lt;/message_type&gt; &lt;/skillz_sdk&gt; &lt;/message&gt;" />
                </request>
             </transaction>
-
-            <thinktime value="30" random="true" />
+            <thinktime value="10" random="true" />
+         </for>
+         <for from="0" to="7" incr="1" var="counter">
+            <transaction name="get_offline_messages">
+               <request subst="true">
+                   <jabber type="raw" ack="no_ack" data="&lt;iq   xmlns=&quot;jabber:client&quot; type=&quot;get&quot; id=&quot;%%ts_user_server:get_unique_id%%&quot;&gt;   &lt;query     xmlns=&quot;http://jabber.org/protocol/disco#info&quot; node=&quot;http://jabber.org/protocol/offline&quot;/&gt;   &lt;/iq&gt;" />
+               </request>
+               <request subst="true">
+                  <jabber type="raw" ack="no_ack" data="&lt;iq   xmlns=&quot;jabber:client&quot; type=&quot;get&quot; id=&quot;%%ts_user_server:get_unique_id%%&quot;&gt;   &lt;query     xmlns=&quot;http://jabber.org/protocol/disco#items&quot; node=&quot;http://jabber.org/protocol/offline&quot;/&gt;   &lt;/iq&gt;" />
+               </request>
+               <thinktime value="5" />
+            </transaction>
          </for>
-         <transaction name="get_offline_messages">
-            <request>
-               <jabber type="raw" ack="no_ack" data="&lt;iq   xmlns=&quot;jabber:client&quot; type=&quot;get&quot; id=&quot;8494d366-45ee-413c-af4f-2ba069e7a41f&quot;&gt;   &lt;query     xmlns=&quot;http://jabber.org/protocol/disco#items&quot; node=&quot;http://jabber.org/protocol/offline&quot;/&gt;   &lt;/iq&gt;" />
-            </request>
-         </transaction>
 
-         <thinktime value="10" random="false" />
+         <thinktime value="3" random="false" />
 
-         <transaction name="presence_broadcast_all">
-            <request>
-               <jabber type="presence:broadcast" show="chat" status="Available to chat" ack="no_ack"/>
-            </request>
-         </transaction>
-         <!-- keep alive -->
-         <thinktime value="120" random="false" />
+         <for from="0" to="600" incr="1" var="counter">
```

Comment:
nit you use `var="i"` for most loops, consistency is good

Reply:
fixed